hack/ocp-e2e-tests-handler.sh

```diff
@@ -59,4 +59,4 @@ while ! oc get pods -n nmstate | grep handler; do sleep 1; done
 # Then wait for them to be ready
 while oc get pods -n nmstate | grep "0/1"; do sleep 1; done
 # NOTE(bnemec): The test being filtered with "bridged" was re-enabled in 4.8, but seems to be consistently failing on OCP.
-make test-e2e-handler E2E_TEST_ARGS="--skip=\"${SKIPPED_TESTS}\" --flakeAttempts=${FLAKE_ATTEMPTS}" E2E_TEST_TIMEOUT=120h
```

Comment:
Ha, how did that h get there? I'm pretty sure this was supposed to be 120 minutes, not hours. :-)

Reply:
Probably missed it on my initial CI PR, when I played around a bit :face_with_spiral_eyes: